### ISSUE TITLE
Genetics: positive traits randomly won't trigger

### ISSUE THREAD

**Issue by SyntaxErrorLine3** [yogstation13/Yogstation#2253]
[Round ID]:  92765
[Round ID]:  92756
[Round ID]:  92764
Several other rounds last night as well

[Reproduction]: while running genetics, nothing out of the ordinary, I get genetic nodes that do not trigger a reaction when I inject them. They are always positive traits, always random, in 92764 it was telekinesis, in 97265 it was xray vision, however there is usually more than 1, 2-3 not showing up seems to be the trend. There is no pattern that I can see in how the powers are found, as I've had the issue with different pulse and duration settings. I have not seen a fellow geneticist have this problem, but in 92765 I had every power except xray, and had the node that I knew xray was on active and stuck both myself and another player into the machine 15-25 times each, we could not make it trigger (I was the only geneticist in the round). In previous rounds the last few days I had the issue but it was not obvious because I always took powers from my fellow geneticists machines who finished before I did, I only noticed that I had nodes that I simply could not trigger a power from, and they were always the positive powers on my fellow geneticists computer. It would appear that if someone else discovers them I can receive and give them, but I cannot give/receive the empty nodes that generate on my own pc, as is shown in the issue I described above in 92765.

There is only one oddity that stands out on this whole issue, that is that my character is a lizard, and the only lizard I have seen working in genetics, all the human coworkers did not have this issue.

**Comment by SyntaxErrorLine3** [comment]
these round ID's may not be right, they seem wacky

**Comment by SyntaxErrorLine3** [comment]
more to add: the round before 18055(rounds ID's don't seem to be sequential?):  Once again, Lizard geneticist, I find strange/mutate early, around node 5-7, then in node 10 I find xray. **Strange does not trigger on node 10**. No other positive powers trigger on me, but I have exactly as many nonresponsive nodes as there are remaining positive powers.

I have 2 buffers: buffer 2 has **only node 10**, which I've confirmed is Xray, firing this node on occupant:delayed several times gives me xray vision. At no time does "you feel strange" trigger. However after a couple minutes I start mutating repeatedly.
I return to the machine and load up buffer 1 with node 10 and every nonresponsive node, firing this on occupant:delay 25ish times gives me only telekinesis, no xray, nothing else. I never mutate on this buffer.

I tried switching back and forth 3 or 4 times throughout the round, every time the results were the same as above.
